packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -65,14 +70,38 @@
   --pf-topology-pipelines__pill-status--m-selected--m-danger--color: var(--pf-global--Color--light-100);
   --pf-topology-pipelines__pill-status--m-selected--m-success--color: var(--pf-global--Color--light-100);
   --pf-topology-pipelines__pill-status--m-selected--m-warning--color: var(--pf-global--Color--light-100);
+  --pf-topology-pipelines__pill-status--m-selected--m-skipped--color: var(--pf-global--Color--light-100);
   --pf-topology-pipelines__pill-status--m-selected--m-in-progress--color: var(--pf-global--Color--light-100);
-  --pf-topology-pipelines__pill-status--m-selected--m-pending--color: var(--pf-global--Color--light-100);
-  --pf-topology-pipelines__pill-status--m-selected--m-running--color: var(--pf-global--Color--light-100);
-  --pf-topology-pipelines__pill-status--m-selected--m-idle--color: var(--pf-global--Color--light-100);
+  --pf-topology-pipelines__pill-status--m-selected--m-pending--color: var(--pf-global--palette--black-1000);
+  --pf-topology-pipelines__pill-status--m-selected--m-running--color: var(--pf-global--palette--black-1000);
+  --pf-topology-pipelines__pill-status--m-selected--m-idle--color: var(--pf-global--palette--black-1000);
 
   --pf-topology-pipelines__pill-badge--fill: var(--pf-global--palette--black-200);
   --pf-topology-pipelines__pill-badge--text--fill: var(--pf-global--palette--black-1000);
 
+  /* status icon, (hidden details) */
+  --pf-topology-pipelines__status-icon--color: var(--pf-global--BorderColor--100);
```

Comment:
All the `--color` variables should have a capital C on color to match our convention; or was that because color was being used for Stroke and Fill sometimes?

Reply:
Yeah, I thought we used `C`olor to mean the `color` property. I'm ok changing it would be more consistent

Reply:
I'm fine with either way, but I'd try to keep it consistent. No need to hold this PR for it since it's an existing thing.